public/index.html

```diff
@@ -26,22 +26,22 @@ <h1 class="header__primary">
         <span class="header__primary--top hdr-text">A Viget Exploration</span>
         <img class="header__primary--btm" src="assets/images/title.svg" alt="improving medical adherence">
       </h1>
-      <img class="header__text-box--pills1" src="assets/images/pills.svg" alt="">
-      <img class="header__text-box--pills2" src="assets/images/yellow-pill.svg" alt="">
-      <img class="header__text-box--pills3" src="assets/images/more-pills.svg" alt="">
+      <img class="header__primary--pills1" src="assets/images/pills.svg" alt="">
+      <img class="header__primary--pills2" src="assets/images/yellow-pill.svg" alt="">
+      <img class="header__primary--pills3" src="assets/images/more-pills.svg" alt="">
```

Comment:
think these class names need to change now that they're not in `header__primary` anymore?

Reply:
lol yep